mappings/net/minecraft/block/OxidizableBlock.mapping

```diff
@@ -1,2 +1,5 @@
 CLASS net/minecraft/class_5812 net/minecraft/block/OxidizableBlock
 	FIELD field_28709 oxidizationLevel Lnet/minecraft/class_5955$class_5811;
+	METHOD <init> (Lnet/minecraft/class_5955$class_5811;Lnet/minecraft/class_4970$class_2251;)V
+		ARG 1 oxidationLevel
```

Comment:
It's said online that oxidation and oxidization are the same. Should we enforce one or the other as a result?

Reply:
Use `oxidization` always.